air/src/execution_step/instructions/ap.rs

```diff
@@ -20,11 +20,17 @@ mod utils;
 use super::ExecutionCtx;
 use super::ExecutionResult;
 use super::TraceHandler;
+use crate::execution_step::execution_context::errors::StreamMapError::FloatMapKeyIsUnsupported;
+use crate::execution_step::execution_context::errors::StreamMapError::MapKeyIsAbsent;
+use crate::execution_step::execution_context::errors::StreamMapError::UnsupportedMapKeyType;
 use crate::execution_step::instructions::ValueAggregate;
 use crate::log_instruction;
 use crate::trace_to_exec_err;
+use crate::CatchableError;
 use crate::JValue;
 use crate::SecurityTetraplet;
```

Comment:
new line to satisfy to our coding standard

Reply:
Fixed